test/e2e/reconciler/scheduling/api_compatibility_test.go

```diff
@@ -0,0 +1,116 @@
+/*
+Copyright 2022 The KCP Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cluster
+
+import (
+	"context"
+	"fmt"
+	"math/rand"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	kcpclientset "github.com/kcp-dev/kcp/pkg/client/clientset/versioned/cluster"
+	apiextensionsclientset "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/rest"
+
+	workloadv1alpha1 "github.com/kcp-dev/kcp/pkg/apis/workload/v1alpha1"
+	kubefixtures "github.com/kcp-dev/kcp/test/e2e/fixtures/kube"
+	"github.com/kcp-dev/kcp/test/e2e/framework"
+)
+
+func TestSchedulingOnSupportedAPI(t *testing.T) {
+	t.Parallel()
+	framework.Suite(t, "transparent-multi-cluster")
+
+	ctx, cancelFunc := context.WithCancel(context.Background())
+	t.Cleanup(cancelFunc)
+
+	source := framework.SharedKcpServer(t)
+	orgClusterName := framework.NewOrganizationFixture(t, source)
+	locationClusterName := framework.NewWorkspaceFixture(t, source, orgClusterName)
+	userClusterName := framework.NewWorkspaceFixture(t, source, orgClusterName)
+
+	kcpClusterClient, err := kcpclientset.NewForConfig(source.BaseConfig(t))
+	require.NoError(t, err)
+
+	firstSyncTargetName := fmt.Sprintf("firstsynctarget-%d", +rand.Intn(1000000))
+	t.Logf("Creating a SyncTarget with no support APIExports and syncer in %s", locationClusterName)
+	_ = framework.NewSyncerFixture(t, source, locationClusterName,
+		framework.WithExtraResources("roles.rbac.authorization.k8s.io", "rolebindings.rbac.authorization.k8s.io"),
```

Comment:
```suggestion
```
the `roles` and `rolebindings` here should not be necessary anymore.
Fixed at the right place in PR https://github.com/kcp-dev/kcp/pull/2379/files#diff-46cd655522637a7621955fb1996a38373f7856662819a02009b111e03848fb13R374